scripts/prepare_sector_network.py

```diff
@@ -2387,6 +2411,79 @@ def add_biomass(n, costs):
             p_nom_extendable=True,
         )
 
+    if biomass_potentials.filter(like="unsustainable").sum().sum() > 0:
+
+        # Create timeseries to force usage of unsustainable potentials
+        e_max_pu = pd.Series(1, index=n.snapshots)
```

Comment:
Can you create directly a pandas dataframe here for the `e_max_pu` instead of creating a series which you convert later into a pandas dataframe?

Reply:
Will be changed.